micro_sam/util.py

```diff
@@ -87,7 +86,7 @@ def get_cache_directory() -> None:
 
     Users can set the MICROSAM_CACHEDIR environment variable for a custom cache directory.
     """
-    default_cache_directory = os.path.expanduser(pooch.os_cache('micro-sam'))
+    default_cache_directory = os.path.expanduser(pooch.os_cache('micro_sam'))
```

Comment:
The name of the cachedir was different from the previous one in `_CHECKPOINT_FOLDER` (see above).
I think `micro_sam` is better than `micro-sam` (because it's the name of the actual python package) but if there is a reason to switch to `micro-sam` instead I would also be fine with it.

Reply:
I don't mind either, so that sounds fine. I don't have any reason for it. I think it's also a good idea to double check there's no remaining `micro-sam` left anywhere else in the code.

Reply:
I quickly checked and this was the only location in the code.
(It's still a bit inconsistent in overall naming, but we can address that later on.)